src/main/java/fi/aalto/cs/apluscourses/intellij/activities/InitializationActivity.java

```diff
@@ -59,18 +60,25 @@ public InitializationActivity(@NotNull Notifier notifier) {
 
   @Override
   public void runActivity(@NotNull Project project) {
+    final var stopwatch = new Stopwatch();
+    logger.info("Starting initialization");
+    logger.info("A+ Courses v{}", BuildInfo.INSTANCE.pluginVersion);
+    var courseVersion = BuildInfo.INSTANCE.courseVersion;
+    logger.info("Course v{}", courseVersion);
+    logger.info("Logs made by Jaakko <3");
```

Comment:
There's no point writing four log entries for what's essentially one event. I suggest "Starting initialization, course version {courseVersion}".

1. "Starting initialization" - can be merged with 3.
2. The "A+ Courses {version}" entry is redundant, since the plugin version is already included in the logger's name.
3. Good, but replace the "v" with "version"
4. umm